src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,14 @@
-export const UserInfo = () => (
-  <>Put the user here</>
-);
+import React from 'react';
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => {
+  if (!user) {
+    return <></>;
+  }
```

Comment:
Do not return fragments, render this component instead only if there is user (make this check in parent)